sig-storage/annual-report-2021.md

```diff
@@ -0,0 +1,164 @@
+  * CSIDriver policy for fsgroup beta 1.20
+    * https://github.com/kubernetes/enhancements/blob/master/keps/sig-storage/1682-csi-driver-skip-permission/kep.yaml
+    * https://kubernetes.io/blog/2020/12/14/kubernetes-release-1.20-fsgroupchangepolicy-fsgrouppolicy/
+  * CSI Migration beta 1.17: https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/625-csi-migration
+    * Azure Disk and vSphere CSI migration beta in 1.19
+    * OpenStack CSI migration beta in 1.18
+    * GCE CSI migration beta 1.17
+    * AWS CSI migration beta 1.17
+    * Azure File alpha 1.15
+  * CSI Volume expansion: beta in 1.16
+    * https://github.com/kubernetes/enhancements/issues/556
+    * https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/284-enable-volume-expansion
+  * CSI inline volume beta 1.16: https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/596-csi-inline-volumes
+
+* Alpha:
+  * Volume populator alpha 1.18: https://github.com/kubernetes/enhancements/blob/master/keps/sig-storage/1495-generic-data-populators/kep.yaml
+  * Storage capacity tracking alpha 1.19: https://github.com/kubernetes/enhancements/blob/master/keps/sig-storage/1472-storage-capacity-tracking/kep.yaml
+  * Generic ephemeral volume alpha 1.19: https://github.com/kubernetes/enhancements/blob/master/keps/sig-storage/1698-generic-ephemeral-volumes/kep.yaml
+  * Volume health alpha 1.19: https://github.com/kubernetes/enhancements/blob/master/keps/sig-storage/1432-volume-health-monitor/kep.yaml
+  * Service account token alpha 1.20: https://github.com/kubernetes/enhancements/blob/master/keps/sig-storage/1855-csi-driver-service-account-token/kep.yaml
+    * https://kubernetes.io/blog/2020/12/18/kubernetes-1.20-pod-impersonation-short-lived-volumes-in-csi/
+
+* Road to alpha:
+  * COSI: https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/1979-object-storage-support
+  * Non-graceful node shutdown: https://github.com/kubernetes/enhancements/pull/1116
+  * Allow Kubernetes to supply pod's fsgroup to CSI driver on mount: https://github.com/kubernetes/enhancements/pull/2323
+  * Volume group: https://github.com/kubernetes/enhancements/pull/1551
+  * Recover from resize failures: https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/1790-recover-resize-failure
+  * SELinux recursive permission handling: https://github.com/kubernetes/enhancements/issues/1710
+  * Volume/Snapshot namespace transfer: https://github.com/kubernetes/enhancements/pull/2326
+  * Volume health (2nd alpha due to re-design): https://github.com/kubernetes/enhancements/pull/2286
+  * Prioritization on volume capacity (sig-scheduling): https://github.com/kubernetes/enhancements/pull/1862
+  * PVC created by statefulset will not be auto removed (sig-apps): https://github.com/kubernetes/enhancements/pull/1915
```

Comment:
Maybe better to point at the submitted KEP for PVC auto-removal? https://github.com/kubernetes/enhancements/tree/master/keps/sig-apps/1847-autoremove-statefulset-pvcs

Reply:
Done